### DISCUSSION TITLE
Turbo Racing Protocol

### DISCUSSION THREAD

**Discussion by Lolo45022** [pascallanger/DIY-Multiprotocol-TX-Module#852] [General]
Hello,

First, thank you very much for all your work. I use Multiprocol module in my Radiomaster TX16S and work great !
I just have a question about Turbo Racing Protocol, I think the name is FHSS VT System.

Does this protocol will arrive on day on the Multiprotocol Module in a future Firmware update ?

Thanks for your answer and again for your work.

Best regards
Laurent

**Comment by pascallanger** [comment]
I've already tried to look at the protocol but I haven't figured out a lot about it... I might try again at some point but don't expect too much.

**Reply by Lolo45022** [reply]
Hello,
Thanks for your answer, I understand.

**Reply by ropalet** [reply]
Is there any update about the protocol?

**Reply by pascallanger** [reply]
https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/902 and https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/875
